app/views/planning_application_mailer/receipt_notice_mail.text.erb

```diff
@@ -4,23 +4,21 @@ Date received: <%= @planning_application.created_at.strftime("%e %B %Y - %H:%M:%
 
 Site address: <%= @planning_application.full_address %>
 Description: <%= @planning_application.description %>
-Application type: <%= @planning_application.application_type %>
+Application type: <%= @planning_application.parsed_application_type %>
 
-What happens next
+What happens next:
 
 We are still checking your application for errors or missing information, and ensuring that all plans and documents comply with all statutory requirements. This is the process for accepting your application and should take less than 14 days. We will contact you during this time if needed.
 
 If your application is valid, then a planning officer may contact you to arrange a site visit.
 
 A planning officer will then make a decision on your application.
 
-If you do not hear from us
-
 If by <%= @planning_application.target_date.strftime("%e %B %Y") %>:
 
-you have not been given a decision in writing
-you have not been told that the application is invalid
-you not agreed in writing to extend the decision period
+- you have not been given a decision in writing
+- you have not been told that the application is invalid
+- you not agreed in writing to extend the decision period
```

Comment:
you have not agreed*

Reply:
oohhh, good catch! I'll put in a PR for this today